src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java

```diff
@@ -15,6 +15,18 @@
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+//new
```

Comment:
This comment will lose its meaning once it is merged into main, as it is no longer "new", please remove.